### DISCUSSION TITLE
Benchmark results for the vision-only tasks

### DISCUSSION THREAD

**Discussion by zzjun725** [PKU-Alignment/safety-gymnasium#107] [Q&A]
I wonder if there are any benchmark results for the vision-only tasks(race, formula-one)? I went through the documentation & paper and it seems no benchmark results related to the vision-only environment. Thanks in advance!

**Comment by muchvo** [comment]
Apologies for the late reply. You can find the benchmark results for the existing vision-only tasks within the [SafeDreamer](https://arxiv.org/abs/2307.07176).

**Reply by zzjun725** [reply]
Got it, thanks!
